src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-space-bettwen;
+  max-width: $max-width-calendar;
+
+  &-day {
+    box-sizing: border-box;
+    width: $day-size;
+    height: $day-size;
+    cursor: pointer;
+    background-color: #eee;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

Reply:
Thank. This exercise broke my brain))